doc/usecases/reward_preview.md

```diff
@@ -0,0 +1,27 @@
+# Use Case: Reward Preview
+
+## Summary
+
+- **Scope:** Reward Preview
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** A user can see a preview of future rewards for his efforts.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  - The user has opened the app and is editing a map.
+  - The user has planted at least one plan on his map.
```

Comment:
As far as I understood Pavlo, I think he meant with reward the real-life 'reward' of seeing the flowers you have grown bloom. Thats why I phrased it so map specific. And while I also think it would be nice to see when the flowers on your map will bloom, maybe we can rephrase this use case here to remove the somewhat misleading term 'reward' and make a seperate use case that includes the user specific rewards.

Reply:
Please add to glossary Blossoms (for gamification) and flowering (if it refers to real plants).

Reply:
```suggestion
  - The user has opened the app and is editing a map.
  - The user has planted at least one plant on his map.
```